### PR TITLE
PR feedback

### PR DESCRIPTION
**Pull request by haveanicedavid**
Figured it was easier to PR than make a bunch of comments in your current one - I'll add some notes

### CODE REVIEW

File: web-registry/src/pages/MyEcocredits/MyEcocredits.tsx [hunk 1]
```diff
@@ -191,6 +190,13 @@ export const MyEcocredits = (): JSX.Element => {
     setTxButtonTitle,
   });
 
+  const sxs = {
+    arrow: {
+      width: theme.spacing(6),
+      height: theme.spacing(6),
+    } as SxProps<Theme>,
+  };
```
Comment (haveanicedavid) [inline web-registry/src/pages/MyEcocredits/MyEcocredits.tsx:198]: example of how reused `sx` props can be extracted. 

Another option would be to wrap the entire `AvailaleCreditsIcon` in `styled`

File: web-components/src/components/buttons/ContainedButton.tsx [hunk 2]
```diff
@@ -10,46 +10,37 @@ interface ContainedButtonProps extends ButtonProps {
...
-          borderColor: theme.palette.secondary.dark,
-        },
-      };
-    }
-
-    if (isGradientBlueGreen) {
-      return {
-        color: theme.palette.primary.main,
+const StyledButton = styled(Button, {
+  shouldForwardProp: prop => prop !== 'colorVariant',
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/buttons/ContainedButton.tsx:14]: Oh yes you're right, forgot about this.

File: web-components/src/components/form/CreateSellOrderForm.tsx [hunk 3]
```diff
@@ -96,17 +93,9 @@ const CreateSellOrderForm: React.FC<FormProps> = ({
...
-                ml: theme.spacing(5),
-                fontWeight: '800',
-                color: 'info.dark',
-                fontSize: theme.typography.textSmall,
-              }}
-            >
+            <Label size="sm" sx={{ mb: 5, ml: 5, color: 'info.dark' }}>
               {sellDenom}
-            </Box>
+            </Label>
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/form/CreateSellOrderForm.tsx:98]: Thanks for letting me know!

File: web-components/src/components/icons/AvailableCreditsIcon.tsx [hunk 4]
```diff
@@ -1,28 +1,25 @@
...
-    width: theme.spacing(14.25),
-    height: theme.spacing(17.25),
-  },
-}));
+import { Theme } from '~/theme/muiTheme';
+import { SxProps } from '@mui/material';
 
 interface IconProps {
-  className?: string;
+  sx?: SxProps<Theme>;
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/icons/AvailableCreditsIcon.tsx:7]: Ok good to know, I find it way easier to style with `sx` anyway so this is a good thing.

**Pull request merged**